Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1367,6 +1367,23 @@
     <!-- YTXT: text for share result card-->
     <string name="submission_status_card_positive_result_share">"Teilen Sie Ihre Zufalls-IDs, damit andere gewarnt werden können."</string>
 
+
+    <!-- XBUT: submission deletion warning button continue -->
+    <string name="submission_deletion_warning_continue_button">"Weiter"</string>
+    <!-- XBUT: submission deletion warning button cancel -->
+    <string name="submission_deletion_warning_cancel_button">"Abbrechen"</string>
+    <!-- XHED: submission deletion warning title -->
+    <string name="submission_deletion_warning_title">"Hinweis"</string>
+    <!-- YTXT: Headline for rapid test submission deletion warning -->
+    <string name="submission_deletion_warning_headline_antigen_test">"Sie haben bereits einen Schnelltest registriert."</string>
```

Comment:
Shouldn't "erfasst"/"registriert" be used consistently? Both are okay, although I think for scanning a QR code, "registrieren" is more precise, if a code is entered manually, "erfassen" is fine.

Reply:
@SabineLoss 
In my understanding "Erfassen" is the process to enter the data into the app, either by scanning QR code or by entering manually. "Registrieren" describes the process, where the data is submitted from app to the backend, and a token is registered on the server and sent back to the app.
Just my 2 cents